src/consensus/malachite/spawn.rs

```diff
@@ -181,9 +182,12 @@ impl MalachiteConsensusActors {
         )
         .await?;
 
+        let mut timeout_config = TimeoutConfig::default();
+        // setting the commit timeout sets up a fixed block production rate.
+        timeout_config.timeout_commit = config.propose_value_delay;
```

Comment:
We already use `propose_value_delay` elsewhere. Should we create a new config parameter for the existing use case? It seems like the 2 places where we use this shouldn't share the same config.

Reply:
Fixed.